.spi.yml

```diff
@@ -0,0 +1,9 @@
+version: 1
+builder:
+   configs:
+   - documentation_targets: ["CareKit", "CareKitUI", "CareKitStore", "CareKitFHIR"]
```

Comment:
Interesting, where do the built docs end up?

Reply:
SPI hosts them on their website, though you can keep building the docs on GitHub pages. You can see the ones I made for ParseCareKit (it even tracks betas): https://swiftpackageindex.com/netreconlab/ParseCareKit/0.13.1/documentation/parsecarekit

Reply:
You don't have to do anything special to release these other than having the `.spi.yml` file in the repo. I simply create my releases using the standard GitHub release process and SPI adds the documents (usually within ~1 hour of the release)

Reply:
Awesome Sven, thanks for looking at this!